package.json

```diff
@@ -30,6 +31,7 @@
     "@types/react": "18.3.12",
     "@types/react-test-renderer": "18.3.1",
     "@typescript-eslint/eslint-plugin": "8.22.0",
+    "eas-cli": "15.0.4",
```

Comment:
This causes a warning, but I really don't want to have to install anything globally.